sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RequestData.cs

```diff
@@ -27,12 +27,16 @@ public RequestData(int version, Activity activity, ref TagEnumerationState monit
                     break;
             }
 
-            Id = activity.Context.SpanId.ToHexString();
-            Name = TraceHelper.GetOperationName(activity, ref monitorTags.MappedTags);
-            Duration = activity.Duration.ToString("c", CultureInfo.InvariantCulture);
+            Id = activity.Context.SpanId.ToHexString().Truncate(SchemaConstants.RequestData_Id_MaxLength);
```

Comment:
Minor: We can skip this truncate. Activity `SpanId` and `TraceId` will be of fixed length.